pmd-java/src/main/resources/category/java/design.xml

```diff
@@ -740,8 +740,10 @@ in each object at runtime.
 <![CDATA[
 //FieldDeclaration
  [@Final= true() and @Static= false()]
-   /VariableDeclarator/VariableInitializer/Expression
-    /PrimaryExpression[not(PrimarySuffix)]/PrimaryPrefix/Literal
+   [VariableDeclarator/VariableInitializer/Expression
+    /PrimaryExpression[not(PrimarySuffix)]/PrimaryPrefix/Literal]
+    [not(preceding-sibling::Annotation/MarkerAnnotation/Name[@Image="Builder.Default"]
```

Comment:
This test could probably be moved to right after the `[@Final= true() and @Static= false()]` for clarity. That way you don't need to put the other path (the one that starts with `VariableDeclarator`) entirely inside a `[...]` predicate. 

Though I noticed the violation is now reported on the FieldDeclaration node whereas it was previously reported on the expression. Probably the correct place to report it is actually the VariableDeclaratorId, since there may be several of them in the same field declaration.

So the expression should most likely have the form 
```java
//FieldDeclaration
[@Final= true() and @Static= false()]
[/* test for lombok.Builder */]
/VariableDeclarator[VariableInitializer/...]/VariableDeclaratorId
```

Reply:
Cheers for the comments @oowekyala , I've made that change and added a test for the

```
class MyClass {
    private final String myField = "something", myOtherField = "somethingElse";
}
```

case to make sure it triggers twice.